modules/blockChain/contractAddresses.ts

```diff
@@ -16,6 +16,11 @@ export const GovernanceToken: ChainAddressMap = {
   [CHAINS.Goerli]: '0x56340274fB5a72af1A3C6609061c451De7961Bd4',
 }
 
+export const StEthToken: ChainAddressMap = {
```

Comment:
We have stETH token addresses declared in the dapp already. See https://github.com/lidofinance/easy-track-ui/blob/develop/modules/blockChain/contractAddresses.ts#L32